pkg/services/sqlstore/migrations/migrations_test.go

```diff
@@ -27,7 +31,7 @@ func TestMigrations(t *testing.T) {
 	mg := NewMigrator(x, &setting.Cfg{})
 	migrations := &OSSMigrations{}
 	migrations.AddMigration(mg)
-	expectedMigrations := mg.MigrationsCount()
+	expectedMigrations := mg.GetMigrationIDs()
```

Comment:
alternatively you could modify `(*Migrator)MigrationsCount()` to optionally not to count not logged migrations:

```
func (mg *Migrator) MigrationsCount(excludeNotLogged bool) int {
	c := 0
	for _, m := range mg.migrations {
		if excludeNotLogged && m.SkipMigrationLog() {
			continue
		}
		c += 1
	}
	return c
}
```

Reply:
Nice! Thanks. I think I will update GetMigrationIDs so next time it will be more clear why it fails